### ISSUE TITLE
Long text is not showing in single line.

### ISSUE THREAD

**Issue by ambujkhanna** [ie-web-notifications/ie-web-notifications.github.io#19]
Hi,

I am facing an issue where i find long text is going outside the notification window.

Expected behavior : If text length is larger than notification box then it should break and show in 2nd/next line as like other browsers.

Please help!!!

Thanks :)

**Comment by ie-web-notifications** [comment]
Yes, it's a long known issue, I will look into it but seems not too soon.

**Comment by ambujkhanna** [comment]
ok..!!! Just try if can provide quick fix for same. 

ThankYou!!!
